modular_skyrat/modules/bodyparts/code/skrell_bodyparts.dm

```diff
@@ -0,0 +1,24 @@
+// Skrell!
+/obj/item/bodypart/head/mutant/skrell
+	icon_greyscale = 'modular_skyrat/modules/bodyparts/icons/skrell_parts_greyscale.dmi'
```

Comment:
I feel like we should probably consider making all of these paths defines, even if they're just defined for that file alone, just so it's easier to maintain in the future if they're ever moved around again.

Reply:
After some considerations, while I'm not enthusiast of the idea - as you said, most of these defines are only used in one file -, it doesn't hurt to try.